packages/cli/src/Interfaces.ts

```diff
@@ -667,7 +667,7 @@ export interface ILicensePostResponse extends ILicenseReadResponse {
 
 export interface JwtToken {
 	token: string;
-	expiresIn: number;
+	expiresInSeconds: number;
```

Comment:
in the context of auth tokens, I've never seen an `expiresIn` that's not in seconds.
While I do see the additional clarity of mentioning seconds here, I don't think we need to rename the property itself.
I've seen an uptick in the number of long descriptive variable nameslately , and I fere like this makes code increasingly harder to read. 
IMO variable names should try to be as simple as possible, and all additional relevant metadata should move to JSDocs.

In this particular case, I'd prefer if renamed this back, and add a more descriptive JSDoc instead. That was anyone inspecting the code will still know that this is in seconds.

Reply:
Happy to change it back, but some background before I do that.

> I've never seen an expiresIn that's not in seconds.

I actually thought exactly the same thing. So I assumed this is in seconds:
https://github.com/n8n-io/n8n/blob/b6d775768f927e69e7aa5f715c99f2fed4eaaa4c/packages/cli/src/auth/jwt.ts#L39-L41

And it is.

I also thought `maxAge` is in seconds:
https://github.com/n8n-io/n8n/blob/b6d775768f927e69e7aa5f715c99f2fed4eaaa4c/packages/cli/src/auth/jwt.ts#L88-L92

Because that's what it is in the browser: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#max-agenumber

And on master we currently pass milliseconds. So I thought I found a bug and fixed it in this PR. 

Now I tried it out locally and it turns out express expects milliseconds and converts it internally:
https://expressjs.com/en/api.html#res.cookie
![image](https://github.com/n8n-io/n8n/assets/927609/39e910b4-46f1-4505-8545-f864d3b9e0d0)

--- 

I'm happy to change it back, but I hope that shined some light on why I thought it may be helpful. If express would call their option `expiresInMilliseconds` I would have not introduced a bug in this PR 😄 

I'll fix the bug first that I have in the PR. Let me know what you think and if you still want me to revert the name change then I will.

Reply:
I was about to report that there was a bug in the duration, glad you've already figured it out.

Reply:
I get what you are saying and I agree that it's important to be clear about what unit a variable uses. My point was that we should use more JSDoc for that, and be leaner with our variable names. I've seen long and descriptive names get out of hand in old projects, and that makes code much harder to read because people put more details about what a variable does by putting all the details in the name.

We should instead [send PRs](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68255) (whenever possible) to add JSDoc on the packages we use to have the docs in our editors
![image](https://github.com/n8n-io/n8n/assets/196144/5cccb754-cf38-46f2-82f2-2c3334a8e7e9)

Reply:
I agree. The JSDoc works well enough for me, so I changed the interface back and documented it via JSDoc.

Good thinking with amending the DT types for express. 
I hope your DT PR goes through 🤞🏾